packages/nimble-storybook/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@ni-private/nimble-storybook",
```

we use the `nimble-` prefix for packages that are nimble specific, I'd say lets do just `storybook` similar to `site`, `performance`, etc.
```suggestion
  "name": "@ni-private/storybook",
```